packages/drivers/odsp-driver/src/odspCache.ts

```diff
@@ -101,7 +101,7 @@ export interface IPersistedCache {
      * @param seqNumber - (reference) sequence number of snapshot. Incoming Ops will start with this number
      * (see updateUsage API).
      */
-    put(entry: ICacheEntry, value: any, seqNumber: number): void;
+    put(entry: ICacheEntry, value: IPersistedCacheValue, seqNumber: number): void;
```

Comment:
This is definitely a breaking change, so we should put that in breaking.md. This will require creating a new schema version and expiring existing cache, we should clearly articulate that.
We should explore keeping this interface simple (i.e. not adding IPersistedCacheValue), but rather use an adapter approach, where we can provide an adapter class that wraps "base" cache interface and exposes IPersistedCacheValue-based APIs. That way host should not care, but internally we always work with IPersistedCacheValue-specific types.

Reply:
Removed this api change. Added an adapter for epoch, so that only the epoch tracker knows that the value it gets  has the epoch. However added the api, which allow removal of all entries for a given docId. Added that in breaking.md. This api seems important as we want to remove all entries for a given docId if epoch mismatches or any other thing happened(so host may also want to use it, for us in case of epoch).